purchase_sale_inter_company/models/stock_picking.py

```diff
@@ -106,6 +106,23 @@ def button_validate(self):
                         dest_company,
                         record.sale_id,
                     )
+        # if the flag is set, block the validation of the picking in the destination company
+        if self.env.company.block_po_manual_picking_validation:
+            for record in self.sudo():
```

Comment:
**question**: what would happen if we were to remove this `sudo()`?

Would we get an access error of some kind?

Reply:
Thank you for pointing this out, there are no access errors so it was just useless